### PR TITLE
DS-920: Update install instructions

### PR DESCRIPTION
**Pull request by outwarped**
**What this PR does / why we need it**:
Update readme with up to date `Add Target Cluster` procedure

**Which issue(s) this PR fixes** (optional)
Closes # [DS-920](https://portworx.atlassian.net/browse/DS-920)

**Special notes for your reviewer**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,55 +4,47 @@ Helm charts for the PDS project.
...
+or:
+
+> <https://staging.pds-dev.io/settings/deployment-targets> - (example link in `staging` environment)
+
+And follow provided steps:
+
+1. Add PDS helm charts:
+
+    ```shell
+    helm repo add pds https://raw.githubusercontent.com/portworx/pds-charts/releases
```
[the remark this answers is not in the document]
Comment (outwarped) [reply README.md:22]: I am not sure what do you mean? Did I miss some details or steps in documentation?

[the remark this answers is not in the document]
Comment (holubales) [reply README.md:22]: Nope, just use the address Jirka provided, the one in the UI is outdated. Sorry about not warning you.

Comment (holubales) [inline README.md:22]:
```suggestion
    helm repo add pds https://portworx.github.io/pds-charts
```

**Review by holubales — APPROVED** [review]
Perfect, thank you for this.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (outwarped) [reply README.md]: covered in #34

**Review by jiri-janousek — APPROVED** [review]
LGTM. You need to update helm chart version to make CI pass.

**Pull request merged**